typings/index.d.ts

```diff
@@ -3077,10 +3083,11 @@ export interface ApplicationCommandNonOptionsData extends BaseApplicationCommand
 export type ApplicationCommandOptionData =
   | ApplicationCommandSubGroupData
   | ApplicationCommandNonOptionsData
+  | ApplicationCommandChannelOptionData
   | ApplicationCommandChoicesData
   | ApplicationCommandSubCommandData;
 
-export type ApplicationCommandOption = ApplicationCommandOptionData & {
+export type ApplicationCommandOption = Omit<ApplicationCommandOptionData, 'ApplicationCommandChannelOptionRawData'> & {
```

Comment:
What is `ApplicationCommandChannelOptionRawData` supposed to be?

Reply:
Oh, that's residual, do need to fix what that was trying to though (the ApplicationCommandOption type shouldn't have channel_types ever)

Reply:
`Omit` is used to remove certain keys from a type. You were likely looking for `Exclude`, which is used for removing a type from a union.

Although, you could just omit the `channel_types` key🤷

Reply:
When it existed it was Exclude, forget to change the key and just put quotes around it 💀. But omitting doesn't work as intended, will figure something out.

Reply:
On second thought, just gonna rewrite this whole union. It causes options to show up on all types when it shouldn't anyways